src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -273,15 +273,15 @@ instance Buildable TxIn where
 data TxOut = TxOut
     { address
         :: !Address
-    , coin
+    , amount
```

Comment:
I have renamed this in order to reuse the same type in post transaction api endpoint https://input-output-hk.github.io/cardano-wallet/api/#operation/postTransaction and implelement correct json:
```
"targets": [
{
"address": "2cWKMJemoBam7gg1y5K2aFDhAm5L8fVc96NfxgcGhdLMFTsToNAU9t5HVdBBQKy4iDswL",
"amount": {}
}
],
```

Reply:
In the end I have abandoned this option. I had to create a separate type in order to wrap `TxOut` fileds with `ApiT` constructor. More about it here https://github.com/input-output-hk/cardano-wallet/pull/111#discussion_r273162373